apps/meteor/client/views/room/contextualBar/Threads/ThreadList.tsx

```diff
@@ -144,7 +145,7 @@ const ThreadList = () => {
 
 				{error && (
 					<Callout mi={24} type='danger'>
-						{error.toString()}
+						{getErrorMessage(error)}
```

Comment:
I saw that this function offers accepts a default value in case it can't parse the error. Might be a good idea to add one here.